README.md

```diff
@@ -10,3 +10,15 @@
 |  7 | TLS | Transport Level Security | Tumor Lysis Syndrome |
 |  8 | PCB | Printed Circuit Board | Post Coital Bleeding |
 |  9 | PID | Process ID | Pelvic Inflammatory Disease |
+|  10 | SAL | Sales Accepted Lead | Sterility Assurance Level |
+|  11 | MQL | Marketing Qualified Lead | Medical Query Language |
+|  12 | PQL | Product Qualified Lead | Physical Quality of Life |
+|  13 | PO | Purchase Order | "per os," (means the medication is taken by mouth) |
+|  14 | SPA | Share Purchas Agreement | Seronegative spondyloarthropathies |
+|  15 | FD | Fully Diluted | Functional Dyspepsia |
+|  16 | TOS | Terms of Service | Thoracic outlet syndrome |
+|  17 | ARR | Annual Recurring Revenue | Aldosterone:Renin Ratio |
+|  18 | IRR | Internal Rate of Return | Infusion-Related Reactions |
```

Comment:
We have check those with our medical team and acronyms are not in use.

Reply:
Check again